controllers/blogController.js

```diff
@@ -1,5 +1,27 @@
-blogs_get = (req, res) => res.render("blog/blogs");
+const Blog = require("../models/blog");
+
+const blogs_get = async (req, res) => {
+  try {
+    const blogs = await Blog.find();
+    res.render("blog/blogs.ejs", {blogs});
+  }
+  catch (err) {
+    console.log(err);
+  }
+}
+
+const blogs_single_get = async (req, res) => {
```

Comment:
Perhaps we could name this, blog_get (singular blog). That way, it might better correspond with the names of the views (blogs.ejs and blog.ejs, one controller for getting all the blogs and another for getting one blog). We can do that later, though. This looks good!